src/WheelsAndGoods.Application/Services/OrdersService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Kirpichyov.FriendlyJwt.Contracts;
+using WheelsAndGoods.Application.Contracts;
+using WheelsAndGoods.Application.Contracts.Services;
+using WheelsAndGoods.Application.Models.Orders;
+using WheelsAndGoods.Core.Exceptions;
+using WheelsAndGoods.DataAccess.Contracts;
+
+namespace WheelsAndGoods.Application.Services
+{
+    public class OrdersService : IOrdersService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IJwtTokenReader _tokenReader;
+        private readonly IApplicationMapper _applicationMapper;
+        public OrdersService(IUnitOfWork unitOfWork, IJwtTokenReader jwtTokenReader, IApplicationMapper applicationMapper)
+        {
+            _unitOfWork = unitOfWork;
+            _tokenReader = jwtTokenReader;
+            _applicationMapper = applicationMapper;
+        }
+
+        public async Task<CreateOrderResponce> CreateOrder(CreateOrderRequest createOrderRequest)
+        {
+            var user = await _unitOfWork.Users.GetById(Guid.Parse(_tokenReader.UserId), false);
+
+            if (user is null)
+            {
+                throw new NotFoundException("Customer not found");
+            }
+
+            var order = _applicationMapper.ToOrder(createOrderRequest);
```

Comment:
Let's add Customer as parameter to the mapper method and assign there. After this we can remove line 37. Assign entity int mapper will be enough (Order.Customer = user), EF will use its CustomerId.